### ISSUE TITLE
打开数字人页面，一直黑屏，显示在加载当中

### ISSUE THREAD

**Issue by VisionaryVibes** [lipku/metahuman-stream#206]
C:\Users\A1835\AppData\Roaming\Python\Python311\site-packages\huggingface_hub\file_download.py:1150: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.    
  warnings.warn(
C:\Users\A1835\AppData\Roaming\Python\Python311\site-packages\transformers\configuration_utils.py:380: UserWarning: Passing `gradient_checkpointin
g` to a config initialization is deprecated and will be removed in v5 Transformers. Using `model.gradient_checkpointing_enable()` instead, or if you are using the `Trainer` API, pass `gradient_checkpointing=True` in your `TrainingArguments`.
  warnings.warn(
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
C:\Users\A1835\AppData\Roaming\Python\Python311\site-packages\transformers\modeling_utils.py:446: FutureWarning: You are using `torch.load` with `
weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle dat
a which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more deta
ils). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during
 unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.
serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  return torch.load(checkpoint_file, map_location="cpu")
D:\anaconda\Lib\site-packages\torch\nn\utils\weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
[INFO] warm up ASR live model, expected latency = 0.560000s
[INFO] warm-up done, actual latency = 0.184837s
Connection state is connecting
Connection state is failed
Connection state is closed

<img width="794" alt="11111" src="https://github.com/user-attachments/assets/8b6cf327-3f87-4f59-9a06-3a3966a204b9">

**Comment by monk-after-90s** [comment]
一样

**Comment by Heaverno** [comment]
俺也一样！

**Comment by beixiang-l** [comment]
同上啊，也没说清楚到底是什么连不上

**Comment by xiaomaofei** [comment]
我也一样，为什么呢

**Comment by ajiader** [comment]
俺也一样

**Comment by l1iquan** [comment]
俺也一样

**Comment by yehx1** [comment]
请问解决了吗

**Comment by yehx1** [comment]
> 请问解决了吗

**Comment by ajiader** [comment]
解决了

**Comment by yehx1** [comment]
你好，方便说一下怎么解决的吗，我也遇到了这个问题

**Comment by pinoccc3** [comment]
我遇到相似的问题，远程服务器部署后打开数字人页面点击start，下方的视频框完全不显示，log一直卡在

`Connection state is connecting`

udp 端口也全部开放了

**Comment by Fucheng-Wu** [comment]
> 我遇到相似的问题，远程服务器部署后打开数字人页面点击start，下方的视频框完全不显示，log一直卡在
> 
> `Connection state is connecting`
> 
> udp 端口也全部开放了

请问解决了吗
